### DISCUSSION TITLE
Display value in lcd

### DISCUSSION THREAD

**Discussion by jojogib** [RomanKubiak/ctrlr#526] [Q&A] [answered]
I want to display different values in different lcds:
value of modulator "modPitch" to lcdlable "lcdPitch".
I cant can't figure out how to do this in lua.
Thank you

**Comment by damiensellier** [comment, marked as answer]
You have to create several uiLabel for that and on mod value change you apply the following method:

```
--
-- Called when a modulator value changes
-- @mod   http://ctrlr.org/api/class_ctrlr_modulator.html
-- @value    new numeric value of the modulator
--
myMethod = function(--[[ CtrlrModulator --]] mod, --[[ number --]] value, --[[ number --]] source)

local ModValue = mod:getModulatorValue() -- to get the current value of your modulator
panel:getLabel("uiLabel"):setText(tostring(ModValue)) -- where uiLabel is the name of your target label

end

```

You can display any data or modulator value to any label with this. Hope it helps!

Damien

**Reply by jojogib** [reply]
Thank you it worked
